api/lib/source/cosmosV0-source.js

```diff
@@ -238,19 +238,28 @@ class CosmosV0API extends RESTDataSource {
       '/staking/pool'
     )
     if (!Array.isArray(response)) return []
-    const proposals = response.map((proposal) => {
-      return this.reducers.proposalReducer(
-        this.network.id,
-        proposal,
-        {},
-        totalBondedTokens
-      )
-    })
+    const proposals = await Promise.all(
+      response.map(async (proposal) => {
+        const [tally, proposer] = await Promise.all([
+          this.query(`/gov/proposals/${proposal.id}/tally`),
```

Comment:
does this need a slash right at the beginning? line 245 does not have one...

Reply:
ah good catch... this one was broken